app/src/main/java/com/example/parkingsystemkotlin/mvp/model/ReservationModel.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.parkingsystemkotlin.mvp.model
+
+import com.example.parkingsystemkotlin.mvp.contract.ReservationContract
+import com.example.parkingsystemkotlin.mvp.model.reservation.DataBase
+import com.example.parkingsystemkotlin.mvp.model.reservation.Reservation
+import java.util.Calendar
+
+class ReservationModel(private val reservationInformationDB: DataBase) : ReservationContract.ReservationModelContract {
+
+    private var isStartDateAndTime: Boolean = true
+    private var startDateAndTime: Calendar? = null
+    private var finishDateAndTime: Calendar? = null
+
```

Comment:
pls use lateinit to reduce the used of nulls

Reply:
done